packages/animations/test/fade_scale_transition_test.dart

```diff
@@ -21,7 +20,6 @@ void main() {
                   onPressed: () {
                     showModal<void>(
                       context: context,
-                      configuration: const FadeScaleTransitionConfiguration(),
```

Comment:
I would not take this line out of the test, since we still want to explicitly test that when `FadeScaleTransitionConfiguration` is used, we have the expected behavior. For example, if the default configuration for `showModal` were to ever change, this test would break

Reply:
Okay, I'll revert this 👍